shuup/admin/modules/orders/views/refund.py

```diff
@@ -75,7 +83,7 @@ def get_context_data(self, **kwargs):
                 extra_css_class="btn-success",
             ),
             URLActionButton(
-                url=reverse("shuup_admin:order.create-full-refund", kwargs={"pk": self.object.pk}),
+                url=reverse("{}".format(full_url), kwargs={"pk": self.object.pk}),
```

Comment:
No need to battle this since full refunds can not be supported for vendors since full refund can be only be done once without bigger core overhaul.